polkadot/node/core/pvf/README.md

```diff
@@ -0,0 +1,56 @@
+# PVF Host
+
+## Running basic tests
+
+The first step is to build the worker binaries:
+
+```sh
+SKIP_WASM_BUILD=1 cargo build --bin polkadot-execute-worker --bin polkadot-prepare-worker
```

Comment:
Might not be needed soon after https://github.com/paritytech/polkadot-sdk/pull/2018?

Reply:
Yep, running `cargo test` would do this automatically if the workers aren't already available or stale.

Reply:
Nice, this just bit me again today where I forgot to rebuild them. I'll try to get to that PR. BTW, how do you tell if they're stale?

Reply:
I was referring to those version checks. Thought they're used for that, are they? :)

Reply:
Yes, you're right. :) What I had in mind is, when testing here we can make the version check more strict. It is not very strict usually (only enforcing the release version of the node), because we don't want to enforce frequent rebuilds for most development, where the PVF workers have not changed. But for these tests specifically, if someone is running them, there is a high likelihood that the workers are being developed and changing. With that in mind, we can additionally take into account the latest git commit hash and rebuild if it's changed. The workers should already have a `--full-version` flag which includes this. Do you think this is reasonable?

Reply:
This would have saved me some time when I forgot to rebuild them [last week](https://github.com/paritytech/polkadot-sdk/pull/2058#discussion_r1374447111).

Reply:
You mean to do a commit hash match instead when given this additinoal flag? Sounds good to me :)

Reply:
Er, I think so. Just to be clear, in #2018 the new function that rebuilds the workers only runs in PVF tests, and there it makes sense to always include the commit hash in the check. So you can pass this flag to the workers when getting their version for the check.